com.unity.template-hd/Assets/SampleSceneAssets/Tutorials/Pages/LightProbes_01.asset

```diff
@@ -35,16 +37,24 @@ MonoBehaviour:
       m_TutorialButtonText: 
     - m_Type: 0
       Title:
-        m_Untranslated: What are Light Probes?
+        m_Untranslated: 
       Text:
-        m_Untranslated: "<b>Light Probes</b> are another component of baked lighting.
-          They are used to apply per-object indirect lighting to dynamic objects
-          which are not meant to receive lighting from Lightmaps.\n\n1. Add a cube
-          in the Scene and set its coordinates to x=30, y=2, z=6. \n\nYou can see
-          that the cube is appropriately lit by the <b>Light Probes</b> and receives
-          green indirect lighting from the wall nearby.\n\n2. Move the cube closer
-          to the tree cage. \n\nNotice how the indirect lighting affecting the cube
-          becomes brighter, as the cube approaches the strong pool of light."
+        m_Untranslated: "<b>Adaptive Probe Volumes</b> is a system that allows dynamic
+          objects to receive indirect lighting. It shares the same goal as <b>Light
+          Probes,</b> but has major differences. Some of them are listed below:\n\n\u2022
+          The probe placement is automatic. Probes no longer need to be placed by
```

Comment:
It might be confusing, it's not only that they 'no longer need' but that 'it's not possible' to place them by hand. Maybe we should rephrase ? 
'Probes are no longer placed by hand' ?